cluster/codon/execute/run.sh

```diff
@@ -0,0 +1,468 @@
+#!/bin/bash
+
+set -e
+
+DEFAULT_QUEUE="standard"
+BIGQUEUE="bigmem"
+RUN=0
+
+STEP1="Step1.drep"
+STEP2="Step2.mash"
+STEP3="Step3.clusters"
+STEP4="Step4.mmseqs"
+STEP5="Step5.gtdbtk"
+STEP6="Step6.annotation"
+STEP7="Step7.metadata"
+STEP8="Step8.postprocessing"
+STEP9="Step9.restructure"
+
+MEM_STEP1="50G"
+MEM_STEP2="10G"
+MEM_STEP3="50G"
+MEM_STEP4="150G"
+MEM_STEP5="500G"
+MEM_STEP6="50G"
+MEM_STEP7="5G"
+MEM_STEP8="5G"
+
+THREADS_STEP1="16"
+THREADS_STEP2="4"
+THREADS_STEP3="8"
+THREADS_STEP4="32"
+THREADS_STEP5="2"
```

Comment:
I think this needs to be changed to 32 to match the other scripts

Reply:
yep... good catch.. sorted in 9ddeb4c5